src/v2/view-builder/views/SuccessView.js

```diff
@@ -5,7 +5,16 @@ import { loc } from 'okta';
 
 const Body = BaseForm.extend({
   title () {
-    return  loc('oie.success.redirect', 'login');
+    // For more info on the API response available in appState, see IdxResponse:
+    // https://github.com/okta/okta-core/blob/master/server/api/src/main/java/com/okta/api/mediation/
+    // services/auth/IdxResponseBuilder.java
+    const {label: appInstanceName, name: appDisplayName} = this.options.appState.get('app');
+    const {identifier: userEmail} = this.options.appState.get('user');
+
+    const appName = appInstanceName ? appInstanceName : appDisplayName;
```

Comment:
I think we should always show appDisplayName only.. appInstanceName can be shown as variable (google_1, salesforce1 et)

Reply:
Product designer on the ticket said to favor `appInstanceName` over `appDisplayName`: https://oktainc.atlassian.net/browse/OKTA-341975?focusedCommentId=1656267
On dev when signing into the IDX org, `appDisplayName` is saasure and `appInstanceName` is Okta Administration. 
`appInstanceName` is favored because it is admin configurable, do you know how I can configure it to ensure this is the right value?

btw I got that label is the instanceName and name is displayName from here: https://github.com/okta/okta-core/blob/master/server/api/src/main/java/com/okta/api/mediation/services/auth/IdxResponseBuilder.java#L1501-L1502

Reply:
If `appInstanceName` is `label`.. then i think this is fine.

You can just go to `Applications` and add different apps with different names. The name you add would appear as  `label`